src/SPC/doctor/item/LinuxMuslCheck.php

```diff
@@ -39,7 +39,9 @@ public function fixMusl(array $distro): bool
         $install_cmd = match ($distro['dist']) {
             'ubuntu', 'debian' => 'apt-get install musl musl-tools -y',
             'alpine' => 'apk add musl musl-utils musl-dev',
-            default => throw new RuntimeException('Current linux distro is not supported for auto-install musl packages'),
+            'rhel' => 'dnf install tar wget git zip bison flex bzip2 cmake patch && wget https://musl.libc.org/releases/musl-1.2.4.tar.gz && tar -zxvf musl-1.2.4.tar.gz && rm musl-1.2.4.tar.gz && cd musl-1.2.4 && ./configure && make -j && make install && cd ..',
```

Comment:
maybe the version shouldn't be hardcoded here, open for suggestions

Reply:
also, since this builds from source, it would work on every unix based os - make this the default branch?

Reply:
edit: openssl apparently requires the full perl installation, gets stuck on openssl build otherwise:

```
[root@localhost openssl]# PKG_CONFIG='/opt/static-php-cli/buildroot/bin/pkg-config' PKG_CONFIG_PATH='/opt/static-php-cli/buildroot/lib/pkgconfig' CC='musl-gcc' CXX='g++' PATH='/opt/static-php-cli/buildroot/bin:/usr/local/musl/bin:/root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin' PKG_CONFIG='/opt/static-php-cli/buildroot/bin/pkg-config' PKG_CONFIG_PATH='/opt/static-php-cli/buildroot/lib/pkgconfig' CFLAGS='' CC='musl-gcc -static -idirafter /opt/static-php-cli/buildroot/include -idirafter /usr/include/  -idirafter /usr/include/x86_64-linux-gnu/ '  ./Configure no-shared zlib --prefix=/ --libdir=lib -static --with-zlib-include=/opt/static-php-cli/buildroot/include --with-zlib-lib=/opt/static-php-cli/buildroot/lib no-legacy linux-x86_64
Can't locate IPC/Cmd.pm in @INC (you may need to install the IPC::Cmd module) (@INC contains: /opt/static-php-cli/source/openssl/util/perl /usr/local/lib64/perl5/5.32 /usr/local/share/perl5/5.32 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 /opt/static-php-cli/source/openssl/external/perl/Text-Template-1.56/lib) at /opt/static-php-cli/source/openssl/util/perl/OpenSSL/config.pm line 19.
BEGIN failed--compilation aborted at /opt/static-php-cli/source/openssl/util/perl/OpenSSL/config.pm line 19.
Compilation failed in require at ./Configure line 23.
BEGIN failed--compilation aborted at ./Configure line 23.
```